source/openapi3/split.yaml

```
@@ -2663,6 +2663,7 @@ components:
         description:
           type: string
           description: User description. Only visible to the payer
+           pattern: '^[ -~p{Emoji}]+$'
```

I think you need to escape the p right?

```
3.1.6 :006 > "hello".match?(/[ -~\p{Emoji}]/)
 => true
3.1.6 :007 > "👍".match?(/[ -~\p{Emoji}]/)
 => true
3.1.6 :008 > "👍".match?(/[ -~p{Emoji}]/)
 => false
```


```suggestion
           pattern: '^[ -~\p{Emoji}]+$'
```